packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -71,7 +71,7 @@ const Goerli: L1Network = {
   rpcURL: rpcURLs[5]
 }
 
-const NitroDevnet: L2Network = {
+const DeprecatedNitroDevnet: L2Network = {
```

Comment:
We can just edit this, we don't need the copy of the old Nitro Devnet config, as it won't work with this version of the SDK anyway.